drivers/i2c/mxc_i2c.c

```diff
@@ -973,6 +973,10 @@ static int mxc_i2c_probe_chip(struct udevice *bus, u32 chip_addr,
 	int ret;
 	struct mxc_i2c_bus *i2c_bus = dev_get_priv(bus);
 
+	/* do not disturb the se050 */
+	if (chip_addr == 0x48)
```

Comment:
would it be better to have the 0x48 be a define that is more descriptive?

Reply:
yes, I was in a bit of a rush and this will need a proper interface upstream (do not chip probe certain devices) so didnt spend too much time on it...

Reply:
Well, I strongly believe that we shouldn't do that here. SE050 should be filtered out somewhere on higher level (as it theoretically can be used not only with NXP SoCs), and best case scenario - DT node for SE050 with `status = "disabled"`.

Yes, we still have special cases where DT information is ignored, like `i2c` command, which can touch any chip whatever it is enabled/disabled in DT, but hopefully you have to execute that command manually :)
If there is no any SE050 driver in U-Boot at all, why even care about that?

Reply:
I think you are missing the background info: we noticed that probing the SE050 (ie, sending an empty message to it) makes it unresponsive once SCP03 has been enabled....so this hack just avoids it.

There are no DT properties that can prevent this from happening (hence why it is a hack :)); for instance, the call to i2c_get_chip_for_busnum will probe but we need to make that call.

Reply:
Yeah, if possible, I would prefer to avoid this hack and have something that can leverage DT until we find out the root cause for the SE050 problem.

Reply:
lets not merge the hack and just have an ad-hoc patch instead on those projects that need it....really this should not happen, I guess NXP probably have something to say. I'll open a ticket